themes/default-bootstrap/js/order-opc.js

```diff
@@ -911,7 +911,7 @@ function multishippingMode(it)
 							cache: false,
 							success: function(data) {
 								$('#cart_summary').replaceWith($(data).find('#cart_summary'));
-								$('.cart_quantity_input').typeWatch({ highlight: true, wait: 600, captureLength: 0, callback: function(val) { updateQty(val, true, this.el); } });
+								$('.cart_quantity_input').typeWatch({ highlight: true, wait: 600, captureLength: 1, callback: function(val) { updateQty(val, true, this.el); } });
```

Comment:
Hi,

With the new version of typewatch, ```this.el``` must be replaced by ```this```.

Best regards